playback.sh

```diff
@@ -3,14 +3,15 @@
 PLAYBACKROOT="$( dirname "$( readlink -f "$0")")/"
 MAKEMSEEDPLAYBACK="${PLAYBACKROOT}make-mseed-playback.py"
 RUNPLAYBACK="${PLAYBACKROOT}playback.py"
+PLAYBACKDATA="${PLAYBACKROOT}data"
```

Comment:
This requires someone to remember the trailing '/'. I think two '/' are not a problem but a missing one is.